app/views/accounts/index.html.erb

```diff
@@ -28,6 +28,14 @@
             Neu erstellte Inhalte sind sofort freigegeben und damit öffentlich lesbar.
           </td>
         </tr>
+        <tr>
+          <td>Extended User</td>
```

Comment:
compared to the other headline, i suggest just `Extended` here

Reply:
Naming of the Role ist "ExtendedUser" als an extension of "User" and is matching the Name in the DropDown and defined by PO in SVAK-8.